SUITesslaApp/SUITesslaApp/SettingsScreen/SettingsScreenView.swift

```diff
@@ -0,0 +1,542 @@
+            .renderingMode(.template)
+            .foregroundColor(Constants.topLockGradient)
+            .frame(width: 30, height: 30)
+            .settingsNeumorfismUnSelectedCircleStyle()
+    }
+
+    private var acSliderNameView: some View {
+        CustomSlider(value: Binding(get: {
+            self.temp
+        }, set: { newValue in
+            guard
+                newValue >= 15,
+                newValue < 31,
+                newValue > self.temp
+            else {
+                guard
+                    self.temp > 15,
+                    self.temp > newValue
+                else { return }
+                withAnimation {
+                    self.completionAmout -= 0.073
+                    self.temp = newValue
+                    self.shadowRadius -= 1
+                }
+                return
+            }
+            withAnimation {
+                self.temp = newValue
+                self.completionAmout += 0.073
+                self.shadowRadius += 1
+            }
+        }),
+                     in: 15...30,
+                     step: 1.0,
+                     track: {
+            Capsule()
+                .foregroundColor(.gray)
+                .frame(width: 200, height: 5)
+        }, fill: {
+            Capsule()
+                .foregroundColor(Constants.topLockGradient)
+        }, thumb: {
+            Image(Constants.thumbImageName)
+                .offset(y: 6)
+        }, thumbSize: CGSize(width: 15, height: 15))
+    }
+
+    private var sliderView: some View {
+        HStack {
+            acSettingsNameView
+            acImageNameView
+                .padding()
+            acSliderNameView
+                .padding()
+        }
+    }
+
+    private var fanSettingsNameView: some View {
+        Text(Constants.fanSettingsNameText)
+            .font(Font.system(size: 13))
+    }
+
+    private var fanImageNameView: some View {
+        Image(systemName: Constants.fanImageName)
+            .renderingMode(.template)
+            .foregroundColor(Constants.topLockGradient)
+            .frame(width: 30, height: 30)
+            .settingsNeumorfismUnSelectedCircleStyle()
+    }
+
+    private var fanSliderNameView: some View {
+        CustomSlider(value: $acSliderValue,
+                     in: 0...255,
+                     track: {
+            Capsule()
+                .foregroundColor(.gray)
+                .frame(width: 200, height: 5)
+        }, fill: {
+            Capsule()
+                .foregroundColor(Constants.topLockGradient)
+        }, thumb: {
+            Image(Constants.thumbImageName)
+                .offset(y: 6)
+        }, thumbSize: CGSize(width: 15, height: 15))
+    }
+
+    private var fanSliderView: some View {
+        HStack {
+            fanSettingsNameView
+            fanImageNameView
+                .padding()
+            fanSliderNameView
+                .padding()
+        }
+    }
+
+    // MARK: - HEATSETTINGS
```

Comment:
```suggestion
    // MARK: - HEATSETTINGS
```
не видела в сайде такой марки